app/views/foreman/unattended/kickstart-katello.erb

```diff
@@ -68,6 +68,10 @@ yum
 dhclient
 ntp
 wget
+<% if host_param('kt_activation_keys') -%>
+<% # if you want to use subscription manager you need to install it now, in %post it would rely on upstream mirror -%>
+subscription-manager
+<% end -%>
```

Comment:
is there any reason to have the if check?  if you're using a katello template, i'd think you would want to install sub-manager

Reply:
ah, i guess for centos 5 and 6 its not available by default.  ACK from me too

Reply:
Also people might want to consume their repos published over HTTP, without sub-man.